install/update.rst

```diff
@@ -19,11 +19,11 @@ Updating Zammad
       for your and your customers safety.
 
       In case you couldn't update for a longer time, please ensure to at least
-      update from **mayor to mayor** version. Big version jumps *may* work but
+      update from **major to major** version. Big version jumps *may* work but
       usually go terribly wrong. As example, expecting the current stable
       version of Zammad being 5.1 and your instance being on Zammad 2.4, your
       path would look like so:
-      ``2.4`` → ``3.0`` → ``4.0`` → ``5.0`` → ``latest stable (5.1)``
+      ``2.4`` → ``3.0`` → ``4.0`` → ``5.0`` → ``latest stable (6.0)``
```

Comment:
This is not correct now, the text says 'being 5.1'. Maybe you can just keep this section as before, or update both places.